test/suite/test_tiered14.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#
+# This is free and unencumbered software released into the public domain.
+#
+# Anyone is free to copy, modify, publish, use, compile, sell, or
+# distribute this software, either in source code form or as a compiled
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import os, random, wiredtiger, wtscenario, wttest
+from wtdataset import TrackedSimpleDataSet, TrackedComplexDataSet
+
+# test_tiered14.py
+#    Test somewhat arbitrary combinations of flush_tier, checkpoint, restarts,
+#    data additions and updates
+class test_tiered14(wttest.WiredTigerTestCase):
+    uri = "table:test_tiered14-{}"   # format for subtests
+
+    auth_token = "test_token"
+    bucket = "mybucket"
+    cachedir = "mybucket-cache"
+    bucket_prefix = "pfx_"
+    extension_name = "local_store"
+
+    multiplier = [
+        ('S', dict(multiplier=1)),
+#        ('M', dict(multiplier=10)),
```

Comment:
While I know that if the S cannot pass, the others don't have a hope either, it would be good to 1. have a comment about what is needed to uncomment them and/or a ticket to remember to turn them on and 2. have a general comment about what we're multiplying here. It took me a while to see that this is used in the test case code and is used nowhere in this test itself. That was confusing and I only figured it out because I saw the other changes in here. Someone looking at just this test won't see that.

Reply:
Added comment with ticket number.